Shell/Options.cpp

```diff
@@ -133,6 +133,15 @@ void Options::init()
     _lookup.insert(&_mode);
     _mode.addHardConstraint(If(equal(Mode::CONSEQUENCE_ELIMINATION)).then(_splitting.is(notEqual(true))));
```

Comment:
Is it worth specialising these `equal(true)` calls to something like `_splitting.isTrue()`?

Reply:
It could be, but I am too lazy to do it at the moment. Looking at `is`

```
template<typename T>
Options::AbstractWrappedConstraintUP Options::OptionValue<T>::is(OptionValueConstraintUP<T> c)
{
    return AbstractWrappedConstraintUP(new WrappedConstraint<T>(*this,std::move(c)));
}
```

what should be the specialization? ;)